fibrechannel/fibrechannel.go

```diff
@@ -1 +1,294 @@
 package fibrechannel
+
+import (
+	"fmt"
+	"github.com/j-griffith/csi-connectors/logger"
+	"io/ioutil"
+	"k8s.io/kubernetes/pkg/util/mount"
+	volumeutil "k8s.io/kubernetes/pkg/volume/util"
+	"os"
+
+	"path"
+	"path/filepath"
+	"strings"
+)
+
+var log *logger.Logger
+
+//Connector provides a struct to hold all of the needed parameters to make our fibrechannel connection
+type Connector struct {
+	VolumeName string
+	TargetWWNs []string
+	Lun        string
+	WWIDs      []string
+}
+
+type FCMounter struct {
```

Comment:
You'll need a docstring on this exported type too.  And then again on each of the exported members of the struct below

Reply:
https://github.com/j-griffith/csi-connectors/pull/6/commits/c41bca1a9c23fee853f503df47e0ac98fecf5830